lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -105,12 +113,19 @@ class LAMPApp extends Script {
     ]);
   }
 
-  addScriptCreateDatbase() {
+  addScriptCreateDatabase() {
     this.script = this.script.concat([
       'echo \'Creating MySQL Database, user and granting access.\'',
-      'mysql -e \'create database ' + this.databaseName + '\'',
-      'mysql -e \'grant all on ' + this.databaseName + '.* to "root"@"localhost"\'',
+      'set +ux', // Don't post variables to build logs.
+      'DATABASE_NAME='+this.databaseName,
+      'DATABASE_USER='+this.databaseUser,
+      'DATABASE_PASS='+this.databasePass,
+      'if [ -e $ASSET_DIR/credentials.sh ]; then source $ASSET_DIR/credentials.sh; fi', // Set db credentials if found.
+      'mysql -e \'create database \'$DATABASE_NAME',
+      'if [ "$DATABASE_USER" != "root" ]; then mysql -e \'create user "\'$DATABASE_USER\'"@"localhost" identified by "\'$DATABASE_PASS\'"\'; fi',
+      'mysql -e \'grant all on \'$DATABASE_NAME\'.* to "\'$DATABASE_USER\'"@"localhost"\'',
       'mysql -e \'flush privileges\'',
+      'set -ux',
```

Comment:
Based on how it's used, the database password isn't a secret and doesn't need to be blocked.  Also, we typically block sensitive data at another point in the process because it's really helpful to see all the commands that occur in the build process. 

We could also print the command with asterisks for the database password.

Reply:
updated

Reply:
I meant that I think we should remove the 'set +ux' above this and in one other place because it will prevent the commands from being written to the logs, right?  We are currently writing this info to the logs.

I think It would be better for end user debugging to filter out secrets and replace them with asterisks instead of not showing the commands.  We can do that later in the process which is what happens for some other data.

Also, the db name, user, and password will be repeated in the Drupal plugin (and probably the wordpress plugin) when writing the settings files.

Reply:
I think I covered this in https://github.com/ProboCI/probo/pull/150/commits/d662c084c5318bff7d53ae4447567f3ab3f9305c

Reply:
Sorry, somehow I missed that change.  Looks good.